Examples/doc-example-apis/Sources/main.swift

```diff
@@ -0,0 +1,201 @@
+    switch result {
+    case .success(let success):
+        print("Successfully deleted the cache")
+    case .error(let err):
+        print("Unable to delete cache: \(err)")
+    }
+}
+
+@available(macOS 10.15, iOS 13, *)
+func example_API_Set(cacheClient: CacheClient, cacheName: String) async {
+    let result = await cacheClient.set(
+        cacheName: cacheName,
+        key: "key",
+        value: "value"
+    )
+    switch result {
+    case .success(_):
+        print("Successfully set item in the cache")
+    case .error(let err):
+        print("Unable to set item in the cache: \(err)")
+    }
+}
+
+@available(macOS 10.15, iOS 13, *)
+func example_API_Get(cacheClient: CacheClient, cacheName: String) async {
+    let result = await cacheClient.get(
+        cacheName: cacheName,
+        key: "key"
+    )
+    switch result {
+    case .hit(let hit):
+        print("Cache hit: \(hit.valueString)")
+    case .miss(_):
+        print("Cache miss")
+    case .error(let err):
+        print("Unable to get item in the cache: \(err)")
+    }
+}
+
+@available(macOS 10.15, iOS 13, *)
+func example_API_Delete(cacheClient: CacheClient, cacheName: String) async {
+    let result = await cacheClient.delete(
+        cacheName: cacheName,
+        key: "key"
+    )
+    switch result {
+    case .success(_):
+        print("Successfully deleted item in the cache")
+    case .error(let err):
+        print("Unable to delete item in the cache: \(err)")
+    }
+}
+
+@available(macOS 10.15, iOS 13, *)
+func example_API_InstantiateTopicClient() {
+    do {
+        let credentialProvider = try CredentialProvider.fromEnvironmentVariable(envVariableName: "MOMENTO_API_KEY")
+        let topicClient = TopicClient(
+            configuration: TopicClientConfigurations.iOS.latest(),
+            credentialProvider: credentialProvider
+        )
+    } catch {
+        print("Unable to create topic client: \(error)")
+    }
+}
+
+@available(macOS 10.15, iOS 13, *)
+func example_API_Publish(topicClient: TopicClient, cacheName: String) async {
+    let result = await topicClient.publish(
+        cacheName: cacheName,
+        topicName: "topic",
+        value: "value"
+    )
+    switch result {
+    case .success(_):
+        print("Successfully published message!")
+    case .error(let err):
+        print("Unable to publish message: \(err)")
+    }
+}
+
+@available(macOS 10.15, iOS 13, *)
+func example_API_Subscribe(topicClient: TopicClient, cacheName: String) async {
+    let result = await topicClient.subscribe(cacheName: cacheName, topicName: "topic")
+    switch result {
+    case .subscription(let subscription):
+        print("Successfully subscribed to topic!")
+        do {
+            for try await message in subscription.stream {
+                print("Received message: \(message)")
+            }
+        } catch {
+            print("Unexpected error while receving message: \(error)")
```

Comment:
I'm getting `Unexpected error while receving message: CancellationError()` here every time I run the program.

Reply:
That's because the timeout is going to cancel the subscribe task so it doesn't hang forever. I think I can catch that specific error and provide a better log output though